web/themes/custom/pineapple/sass/base/_base.scss

```diff
@@ -0,0 +1,20 @@
+.block-system-main-block {
+ width: $gutter-size;
+  margin: 0 auto;
+
+  .breakout {
+    margin-right: -50vw;
+    margin-left: -50vw;
+    right: 50%;
+    left: 50%;
+    position: relative;
+    width: 100vw;
+  }
```

Comment:
This code should not be inside the `.block-system-main-block` container, you should remove that block system code

Reply:
done